en-us/multimc-auto-update.md

```diff
@@ -34,11 +34,46 @@ It does _not_ auto-update the individual mods outside of what is set in the the
 
 You'll have to download a new version of the pack that is specific to that Minecraft version.
 
-### Can I ignore some of the mods?
+## Can I ignore some of the mods?
 
-There is no official procedure for this yet, but RaptaG has made a tutorial and an improved version of the [Remty5's workaround](https://github.com/Fabulously-Optimized/fabulously-optimized/issues/81). The user now only has to replace the instance parameters and list the mods that will be disabled. It has been confirmed to work on Linux, may also work for MacOS. Volunteers are free to port it to Windows as well.
+There is no official procedure for this yet, but [RaptaG](https://github.com/RaptaG) and [Ultrasonic1209](https://github.com/Ultrasonic1209) have made tutorials and scripts for an improved version of the [Remty5's workaround](https://github.com/Fabulously-Optimized/fabulously-optimized/issues/81). The user now only has to replace the instance parameters and list the mods that will be disabled. 
+
+If you have any problems, you can ask for support in our [Discord server](https://discord.gg/yxaXtaQqdB).
+
+### Windows
```

Comment:
> `powershell` does not exist in Windows versions older than 10 (to my knowledge)
Powershell is bundled with Windows 7. While it can be installed on systems as old as XP, I don't think it's worth it to mention operating systems that are over a decade old.